features/in_onderzoek.feature

```diff
@@ -739,38 +739,38 @@ Functionaliteit: in onderzoek
-      | hele categorie bij vastgesteld niet-Nederlander         |               | V                                   | 19710417              | 040000 | true | true          | true        |
-      | hele categorie bij behandeld als Nederlander            |               | B                                   | 19520831              | 040000 | true | true          | true        |
-      | nationaliteit                                           | 0052          |                                     | 19560317              | 040510 | true | true          |             |
-      | onbekende nationaliteit                                 | 0000          |                                     | 19560317              | 040510 | true | true          |             |
-      | reden opname                                            | 0052          |                                     | 19560317              | 046310 |      |               | true        |
-      | bijzonder Nederlanderschap vastgesteld niet-Nederlander |               | V                                   | 19710417              | 046510 | true |               |             |
-      | bijzonder Nederlanderschap behandeld als Nederlander    |               | B                                   | 19710417              | 046510 | true |               |             |
+      | gegeven in onderzoek                                    | nationaliteit | aanduidingBijzonderNederlanderschap | nationaliteit type         | datumIngangGeldigheid | waarde | type in onderzoek | nationaliteit in onderzoek | redenOpname in onderzoek |
+      | hele categorie nationaliteit                            | 0052          |                                     | Nationaliteit              | 19560317              | 040000 | true              | true                       | true                     |
+      | hele categorie bij onbekende nationaliteit              | 0000          |                                     | NationaliteitOnbekend      | 00000000              | 040000 | true              |                            | true                     |
+      | hele categorie bij vastgesteld niet-Nederlander         |               | V                                   | VastgesteldNietNederlander | 19710417              | 040000 | true              |                            | true                     |
+      | hele categorie bij behandeld als Nederlander            |               | B                                   | BehandeldAlsNederlander    | 19520831              | 040000 | true              |                            | true                     |
+      | nationaliteit                                           | 0052          |                                     | Nationaliteit              | 19560317              | 040510 | true              | true                       |                          |
```

Comment:
Moet de kolom 'type in onderzoek' hier niet leeg zijn? Als je het polymorfe type 'Nationaliteit' kiest komt de property 'type' immers niet voor in de property 'inOnderzoek'.

Reply:
Nee. Het type wordt bepaald door 2 gegevens: nationaliteit en aanduidingBijzonderNederlanderschap. Wanneer één van die twee in onderzoek staat, is ook type in onderzoek. In dit geval staat nationaliteit in onderzoek en dus moet ook type in onderzoek worden opgenomen.